modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/configReducer.es.js

```diff
@@ -256,6 +256,17 @@ export default (state, action) => {
 				}
 			});
 
+			if (action.dataObject.id === state.dataObject.id) {
```

Comment:
Hi @marcosapmf, adding this code makes no difference to the final result. The problem was really that we were not updating the data object at the time of creating the first form view of a recent created data object, thank you so much for identifying this :)